src/sources/aws_s3/sqs.rs

```diff
@@ -592,6 +600,21 @@ impl IngestorProcess {
     }
 }
 
+#[derive(Clone, Debug, Deserialize)]
+enum Event {
+    TestEvent(S3TestEvent),
+    Event(S3Event),
+}
+
+// https://docs.aws.amazon.com/AmazonS3/latest/userguide/how-to-enable-disable-notification-intro.html
```

Comment:
I'm not entirely sure how this link relates to the test event - it seems like the `notification-content-structure.html` below also covers this event. Perhaps that comment should just get pulled up to the new `Enum`?